web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -417,23 +416,20 @@ public boolean supportsPropertyName(String propertyName) {
 		return OpenmrsConstants.GP_SEARCH_DATE_DISPLAY_FORMAT.equals(propertyName);
 	}
 	
-	public static void setDefaultDateCache(String defaultDateCache) {
```

Comment:
@Nuwantha same here: I think its best you exclude these for now from your PR. I am unsure if we should simply remove them.